src/KeyVault/KeyVault.Test/Scripts/ControlPlane/KeyVaultManagementTests.ps1

```diff
@@ -693,20 +693,32 @@ function Test-ModifyAccessPolicyEnabledForDiskEncryption {
 }
 
 function Test-ModifyAccessPolicyNegativeCases {
-    Param($existingVaultName, $rgName, $objId)
+    $objId = "" # INTENTIONAL
+    $rgName = getAssetName
+    $vaultName = getAssetName
+    $rgLocation = Get-Location "Microsoft.Resources" "resourceGroups" "West US"
+    $vaultLocation = Get-Location "Microsoft.KeyVault" "vault" "West US"
+    New-AzResourceGroup -Name $rgName -Location $rgLocation
```

Comment:
The creation/delete of resources are in test scripts now. It's OK to remove usage of the management clients in c# code